app/views/users/unlocks/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend unlock instructions</h2>
+
+<%= form_for(resource, as: resource_name, url: unlock_path(resource_name), html: { method: :post }) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Resend unlock instructions" %>
+  </div>
+<% end %>
+
+<%= render "users/shared/links" %>
```

Comment:
The only time we import devise views is because we want to modify the styling. If you, however, do not want to modify the styling then there is no need to import all the views. As devise works out of the box. So in the future, only import the views if there is a need to overwrite the styling.